src/panels/config/automation/trigger/types/ha-automation-trigger-persistent_notification.ts

```diff
@@ -0,0 +1,117 @@
+import memoizeOne from "memoize-one";
+
+import { css, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import "../../../../../components/ha-button-menu";
+import "../../../../../components/ha-check-list-item";
+import "../../../../../components/ha-icon-button";
+import "../../../../../components/ha-textfield";
+import { PersistentNotificationTrigger } from "../../../../../data/automation";
+import { HomeAssistant } from "../../../../../types";
+import type { TriggerElement } from "../ha-automation-trigger-row";
+import type { LocalizeFunc } from "../../../../../common/translations/localize";
+import type { SchemaUnion } from "../../../../../components/ha-form/types";
+
+const DEFAULT_UPDATE_TYPES = ["added", "removed"];
+const DEFAULT_NOTIFICATION_ID = "";
+
+@customElement("ha-automation-trigger-persistent_notification")
+export class HaPersistentNotificationTrigger
+  extends LitElement
+  implements TriggerElement
+{
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public trigger!: PersistentNotificationTrigger;
+
+  @property({ type: Boolean }) public disabled = false;
+
+  private _schema = memoizeOne(
+    (localize: LocalizeFunc) =>
+      [
+        {
+          name: "notification_id",
+          required: false,
+          selector: { text: {} },
+        },
+        {
+          name: "update_type",
+          type: "multi_select",
+          required: false,
+          options: [
+            [
+              "added",
```

Comment:
I had to write these out because it wanted read only

Reply:
Then I guess I wasn't so far of with my innitial attemt to localize this.
Reworked it based on @bramkragten's [comment](https://github.com/home-assistant/frontend/pull/16967#discussion_r1235372639).